tests/Application/templates/bundles/SyliusAdminBundle/PaymentMethod/_mollieMethodsForm.html.twig

```diff
@@ -16,87 +16,96 @@
-                                {{ form_help(methodForm.paymentType) }}
-                            {% else %}
-                                {{ form_row(methodForm.paymentType) }}
-                                {{ form_help(methodForm.paymentType) }}
-                            {% endif %}
+                    <div class="field">
+                        <div id="country-excluded_{{ loop.index0 }}">
+                            {{ form_row(methodForm.countryLevel_excluded, { 'attr': {'class': 'fluid search'} }) }}
                         </div>
-                        <div class="four wide field">
-                            {{ form_row(methodForm.orderExpiration) }}
+                    </div>
+                    <div class="field">
+                        <div class="fields">
+                            <div class="twelve wide field">
+                                {% if methodForm.vars.value.methodId == mealvouchersId or
+                                    methodForm.vars.value.methodId == klarnaPayLater  or
+                                    methodForm.vars.value.methodId == klarnaSliceIt %}
+                                    {{ form_row(methodForm.paymentType, {'attr': {'disabled': 'disabled'}}) }}
+                                    {{ form_help(methodForm.paymentType) }}
+                                {% else %}
+                                    {{ form_row(methodForm.paymentType) }}
+                                    {{ form_help(methodForm.paymentType) }}
+                                {% endif %}
+                            </div>
+                            <div class="four wide field">
+                                {{ form_row(methodForm.orderExpiration) }}
+                            </div>
                         </div>
                     </div>
-                </div>
-                <div class="field">
-                    {{ form_widget(methodForm.paymentDescription) }}
-                    <div id="payment_description_{{ name }}">
-                        {{ methodForm.paymentDescription.vars.help|trans|raw|nl2br }}
+                    <div class="field">
+                        {{ form_widget(methodForm.paymentDescription) }}
+                        <div id="payment_description_{{ name }}">
+                            {{ methodForm.paymentDescription.vars.help|trans|raw|nl2br }}
+                        </div>
                     </div>
+                    <h4 class="ui horizontal divider header">
+                        {{ 'bitbag_sylius_mollie_plugin.ui.payment_fee'|trans }}
+                    </h4>
+                    {{ form_row(methodForm.paymentSurchargeFee) }}
                 </div>
-                <h4 class="ui horizontal divider header">
-                    {{ 'bitbag_sylius_mollie_plugin.ui.payment_fee'|trans }}
-                </h4>
-                {{ form_row(methodForm.paymentSurchargeFee) }}
             </div>
         {% else %}
```

Comment:
it works? where is the start of the block?

Reply:
It works